scripts/mergeModels.sh

```diff
@@ -30,19 +30,20 @@ mergeFiles() {
     cat ${TEMP} | sed '/^$/N;/^\n$/D' > ${MODELS}
 }
 
-mergeFilesInReleaseDir() {
+mergeFilesInDir() {
     for sdk in `ls ${RELDIR}`; do
 	MODELDIR=${RELDIR}/${sdk}/models
 	if [ ! -d ${MODELDIR} ]; then
 	    echo "Bail, this directory does not seem right: ${MODELDIR}"
-	    exit 1
+	    continue
 	fi
 	pushd ${MODELDIR}    
 	if [ -f ${MODELS} ]; then
 	    echo "${sdk} has ${MODELS} already generated"
-	else
-	    mergeFiles
+        echo "Removing ${MODELS}"
+        rm -f ${MODELS}
```

Comment:
Nice!!! 

Nit: can we align the indentation? It appears line 42 is indented more than lines 43 and 44, but they should all have the same indentation

Reply:
fixed